### DISCUSSION TITLE
Alternative Letter Styles for Specific Characters in Inter Font

### DISCUSSION THREAD

**Discussion by FabriceSpahn** [rsms/inter#760] [Q&A] [answered]
I’m currently working on a project using the Inter font, and I thought it would be great if Inter offered alternate letter styles. Many fonts nowadays provide alternatives for specific characters like ‘R, G, M, J, Y, r, y, g, a,’ which add flexibility in design. I know there are already some, but there could be even more options to promote variety.

Would it be possible to consider adding such options in future updates?

I’ve added an example.

<img width="1037" alt="image" src="https://github.com/user-attachments/assets/f436ebe1-1d66-42bf-97bd-ae4db3db87a6">

**Comment by rsms** [comment, marked as answer]
The G with a spur is already available in Inter.
Stylistic changes like this is best done as a fork of Inter; many have done so. The most recent example is [Guggenheim Sans by Pentagram](https://www.pentagram.com/work/guggenheim-3) 

I'm saying no to style additions like these mostly on principle: If I'm adding every possible stylistic alternative, what even is Inter? Also, every alternate style imposes a tax on the future as complexity of the source increases quite a bit.
